internal/receiver/smartagentreceiver/converter/traces.go

```diff
@@ -24,12 +25,51 @@ import (
 	"go.uber.org/zap"
 )
 
+const (
+	sfxDataSourceKey         = "data_source_ip"
```

Comment:
nit: I believe we can use github.com/signalfx/signalfx-agent/pkg/core/common/constants.DataSourceIPKey instead

Reply:
For some reason I thought it was internal and not exposed.